packages/cli/src/commands/experimental/templates/realtime.ts.template

```diff
@@ -1,14 +1,4 @@
-import { createPubSub, InMemoryLiveQueryStore } from '@redwoodjs/graphql-server'
+import { Redis } from 'ioredis'
```

Comment:
I was able to improve the way pubSub gets typed in each subscription and service which negated the need to return and type the pubSub in all cases, ie. the intersection of each subscription payload type.

And now the useRedwoodRealtime can be smarter about how the LQ and Sub store is created -- in-memory or using Redis.

Thus the realtime lib, doesn't have to do much anymore.

I wonder if I should just rename this to a redis client lib -- but didn't want to collide with any current app redis clients.